src/pathogena/cli.py

```diff
@@ -424,6 +424,13 @@ def validate(upload_csv: Path, *, host: str | None = None) -> None:
     default=lib.DEFAULT_METADATA["district"],
     show_default=True,
 )
+@click.option(
+    "--pipeline",
```

Comment:
Can this be `--specimen_organism` instead? Otherwise it doesn't match the name in the resultant CSV which is odd

Might need a little thought on if `covid` is the best value here as I chose it without any input on if it's the best. Especially as the myco pipeline runs on `mycobacteria`, does this need to be `coronavirus` or similar?

Reply:
Thanks. First point is addressed. Will find time to discuss the second.